Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -630,6 +630,30 @@ TEST_CASE(test_tiff_ccitt3_fill)
     EXPECT_EQ(frame.image->get_pixel(5, 3), Gfx::Color::NamedColor::White);
 }
 
+TEST_CASE(test_tiff_ccitt3_2d)
+{
+    auto file = TRY_OR_FAIL(Core::MappedFile::map(TEST_INPUT("tiff/ccitt3_2d.tiff"sv)));
```

Comment:
can you say how you created the test images?

Reply:
For once I thought to write it, but I ended fixing up the commit where I put these tests, loosing the explanation in the process :sweat_smile: